project/project.json5

```diff
@@ -9,120 +9,43 @@
             version: '1.0.0',
             required_dimensions: {
                 single_dimensional: {
-                    subsector: {
-                        base: [
-                            "unspecified",
-                        ],
-                    },
-                },
-                multi_dimensional: [
-                    {
-                        sector: {
-                            base: [
-                                "com",
-                            ],
-                        },
-                        metric: {
-                            subset: [
-                                {
-                                    name: 'End Uses by Sector',
-                                    selectors: [
-                                        'commercial_end_uses',
-                                    ],
-                                },
-                            ],
-                        },
+                    sector: {
+                        base: ['res', 'com'],
                     },
-                    {
-                        sector: {
-                            base: [
-                                'res',
-                            ],
-                        },
-                        metric: {
-                            subset: [
-                                {
-                                    name: 'End Uses by Sector',
-                                    selectors: [
-                                        'residential_end_uses',
-                                    ],
-                                },
-                            ],
-                        },
+                    subsector: {
+                        base: ['unspecified'],
                     },
-                ],
-            },
+                    metric: {
+                        subset: [
+                            {
+                                name: 'End Uses by Sector',
+                                selectors: ['residential_end_uses', 'commercial_end_uses'],
+                            },
+                        ]
+                    }
+                }
+            }
         },
         {
-            dataset_id: 'decarb_2023_transport',
```

Comment:
Something went wrong with your commit with message = "changes to use time wrapping feature”. The TEMPO dataset section is now gone. The subset dimension with dimension_query_name = “subsectors_by_sector” is gone.